docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -4,42 +4,40 @@
 Low Birth Weight and Short Gestation
 ====================================
 
-Risk Description
--------------------
+Risk exposure overview
+++++++++++++++++++++++
 
-Case definition in GBD 2017
-+++++++++++++++++++++++++++
+Risk exposure description in GBD 2017
+-------------------------------------
+
+The meaning of the “low birth weight” and “short gestation” in GBD have subtle definitional differences
+compared to other usages of “low birth weight” and “short gestation” in literature. The term “low birth
+weight” has historically been used to refer to birth weight (BW) less than 2500 grams. However, because
+the goal of the GBD risk factors analysis is to quantify the entirety of attributable burden due to each
+risk factor, the GBD definition of “low birth weight” therefore refers to all birth weight below the
+Theoretical Minimum Risk Exposure Level (TMREL) for birth weight. Likewise, new-borns have been
+typically been classified into gestational age (GA) categories of “extremely preterm” (<28 weeks of
+gestation), “very preterm” (28-<32 weeks of gestation), and “moderate to late preterm” (32-<37 weeks
+of gestation). “Short gestation” in GBD refers to all gestational ages below the gestational age TMREL.
+
+Exposures and relative risks for the GBD Low birth weight and short gestation risk factors are divided
+into joint 500-gram birth weight and 2-week gestational age combinations. The lowest risk overall 500-
+gram/2-week bin is the overall TMREL. The univariate TMRELs vary with GA and BW. The lowest risk GA
```

Comment:
The GBD 2017 methods appendix notes that LBWSG categories 53, 54, and 56 were used as the TMREL, but the get_draws RRs indicate that categories 55 and 56 were used as the TMREL. Helena told us that the risk factor methods appendix was wrong and that categories 55 and 56 were the correct TMRELs. This might be good to note here! (Maybe you do later, so disregard if so!)

Reply:
yap! its noted in the TMREL section below